src/wandbot/ingestion/prepare_data.py

```diff
@@ -778,7 +762,7 @@ def extract_tags(source_url: str, report_content: str) -> List[str]:
         tags = list(set([tag.title() for tag in tags]))
 
         if ("wandb.log" or "wandb.init") in report_content:
-            tags.append("contains-wandb-code")
+            tags.append("Contains Wandb Code")
```

Comment:
can we keep these tags as lowercased, hyphenated? it'll help with passing tags via endpoints i think

Reply:
The reason for this was with the unhyphenated text it's easier to do metadata filter using `contains` which is not possible with the hyphenated text. The API can take tag filters than are unhyphenated.